content/GPR/GPR_fundamental_principles.rst

```diff
@@ -0,0 +1,402 @@
+.. _GPR_fundamental_principles:
+
+Basic Principles
+****************
+
+Here, we present the fundamental principles which govern ground penetrating radar (GPR) signals.
+As you will see, many of the basic fundamentals which are used to describe seismic methods can also be applied to GPR.
+As we mentioned earlier, the source of the GPR system sends a pulse of high-frequency electromagnetic waves (radiowaves) into the Earth.
+And as these radiowaves propagate through the Earth, they are distorted due to the distribution of subsurface electromagnetic properties (:math:`\sigma , \; \mu` and :math:`\varepsilon`).
+
+When radiowaves come into contact with an interface (a boundary defined by an abrupt change in the Earth's electromagnetic properties), portions of the incoming radiowaves can be reflected, transmitted and/or refracted.
+This is illustrated in the figure below.
+The reflection, transmission and refraction of the radiowaves depends on the electromagnetic properties defining each side of the interface as well as the incident angle of the incoming radiowave signal.
+
+For the purposes of GPR, the Earth may be thought of as a set of homogeneous regions separated by interfaces.
+Using signals measured by the receivers, the goal of GPR is to define these interfaces and thus gain information about structures under the Earth's; which can be naturally occurring or man-made.
+
+
+
+.. figure:: images_new/GPR_schematic_example.jpg
+		:align: center
```

Comment:
In the future, please use 4 spaces to indent and keep the alignment of modifiers (eg `:align: center` and the caption the same. If you are using sublime text, you can do this by setting your sublime preferences or by clicking on the bottom right where it says "spaces: 4" . See for example: http://tutorials.simpeg.xyz/content/gettingstarted.html#text-editors